ordering/containers/OrderSummary.js

```diff
@@ -0,0 +1,11 @@
+import h from 'react-hyperscript'
+import { compose } from 'recompose'
+
+// Only mock data at the stage, nothing in state or feathers.
+import getOrderSummaryProps from '../getters/getOrderSummaryProps'
+import OrderSummary from '../components/OrderSummary'
+
+export default compose(
```

Comment:
my thinking is that we will have to connect to redux state in this container - as we access it from a new route, meaning it's possible no data is loaded yet. so IMO it'd look something like the other containers that use `feathers-action-react` do, with `query` and `shouldQueryAgain` functions to fetch all the data needed.

Reply:
Yea, agreed. I did leave that out waiting to link with the feathers/database ticket. Happy to add that back, if that is possible without interacting with the database.